samples/remote_attestation/README.md

```diff
@@ -60,7 +60,7 @@ Note: `local report` is the same as an `Intel SGX report`, while the `remote rep
 
 In a typical Open Enclave application, it's common to see multiple enclaves working together to achieve common goals. Once an enclave verifies the counterpart is trustworthy, they can exchange information on a protected channel, which typically provides confidentiality, integrity and replay protection.
 
-This is why instead of attesting an enclave to a remote (mostly cloud) service, this sample demonstrates how to attest two enclaves to each other by using Open Enclave APIs `oe_get_report` and `oe_verify_report` which takes care of all remote attestation operations.
+This is why instead of attesting an enclave to a remote (mostly cloud) service, this sample demonstrates how to attest two enclaves to each other by using Open Enclave APIs `oe_get_evidence` and `oe_verify_evidence` which takes care of all remote attestation operations.
 
 To simplify this sample without losing the focus in explaining how the remote attestation works, host1 and host2 are combined into one single host to eliminate the need for additional socket code logic to deal with communication between two hosts.
 
```

Comment:
github won't let me add a comment on line 73, but the comment is that images/remoteattestation_service.png has the old name RemoteReportWithPKey and needs to be updated.